eservice-descriptors-archiver/src/main/scala/it/pagopa/interop/eservicedescriptorsarchiver/Main.scala

```diff
@@ -28,32 +29,38 @@ object Main extends App {
   private val readModelService: MongoDbReadModelService = new MongoDbReadModelService(
     ApplicationConfiguration.readModelConfig
   )
-  private val sqsHandler: SQSHandler                    =
-    SQSHandler(ApplicationConfiguration.archivingPurposesQueueUrl)(blockingEC)
+
+  private val sqsHandlerConfig: SQSHandlerConfig =
+    SQSHandlerConfig(
+      queueUrl = ApplicationConfiguration.archivingEservicesQueueUrl,
+      visibilityTimeout = ApplicationConfiguration.visibilityTimeout
+    )
+
+  private val sqsHandler: SQSHandler =
+    SQSHandler(sqsHandlerConfig)(blockingEC)
 
   private val job: JobExecution = JobExecution(readModelService, CatalogProcessServiceImpl(blockingEC))
 
   logger.info("Starting eservice versions archiver job")
 
-  private val execution: Future[Unit] = sqsHandler.processAllRawMessages(
-    ApplicationConfiguration.maxNumberOfMessagesPerFile,
-    ApplicationConfiguration.visibilityTimeout
-  ) { (messages: List[String], receipts: List[String]) =>
-    logger.info(s"Saving ${messages.size} messages")
-    for {
-      _ <- Future.traverse(messages)(job.archiveEservice)
-      _ <- Future.traverse(receipts)(sqsHandler.deleteMessage)
-    } yield ()
-  }
-
-  private val init: Future[Unit] = execution
+  private val execution: Future[Unit] =
+    sqsHandler.processAllRawMessages(ApplicationConfiguration.maxNumberOfMessagesPerFile) {
```

Comment:
I still do not get what `maxNumberOfMessagesPerFile` is for, considering that this job does not store files